scripts/check.sh

```diff
@@ -1,8 +1,15 @@
 #!/bin/sh
```

Comment:
The meanings of the `set` options are particular to Bash, so we should change the shebangs to `#!/usr/bin/env bash`.

Reply:
Done.